.buildkite/commands/build-for-testing.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+set -e
+
+echo "--- :rubygems: Setting up Gems"
+restore_cache "$(hash_file .ruby-version)-$(hash_file Gemfile.lock)"
+gem install bundler
+bundle install
+save_cache vendor/bundle "$(hash_file .ruby-version)-$(hash_file Gemfile.lock)"
+
+echo "--- :cocoapods: Setting up Pods"
+
+# Caching the specs repos and global pod cache can dramatically improve Pod times
+restore_cache "$BUILDKITE_PIPELINE_SLUG-specs-repos"
+restore_cache "$BUILDKITE_PIPELINE_SLUG-global-pod-cache"
+
+restore_cache "$(hash_file Podfile.lock)"
+bundle exec pod install || bundle exec pod install --repo-update --verbose
+save_cache Pods "$(hash_file Podfile.lock)"
+
+save_cache ~/.cocoapods "$BUILDKITE_PIPELINE_SLUG-specs-repo"
+save_cache ~/Library/Caches/CocoaPods/ "$BUILDKITE_PIPELINE_SLUG-global-pod-cache"
+
+echo "--- :writing_hand: Copy Files"
+cp -v fastlane/env/project.env-example .configure-files/project.env
```

Comment:
Is copying the example file in the project folder necessary, maybe something left over to ensure old checkouts still work? Or can we remove it?

Reply:
Yeah, just for backward compatibility. We can probably remove it before too long 🙂

Reply:
Noted. 🎉